### DISCUSSION TITLE
Nodes aren't connecting to the primary server

### DISCUSSION THREAD

**Discussion by temitopek** [puppetlabs/community#58] [Puppet Enterprise]
i just signed the certificates to my nodes from the enterprise and yet my nodes aren't connected except for the primary server    
![image](https://github.com/puppetlabs/community/assets/65440596/24899cf3-eb71-4c6e-89a3-1ec67f5ef57a)

**Comment by bastelfreak** [comment]
Did you run puppet agent on them? Are they properly configured with the pxp-agent?

**Comment by bastelfreak** [comment]
Also 'localhost.localdomain' looks really wrong. If that's your primary you should configure your FQDN properly.
